addons/mail/static/src/components/message/message.xml

```diff
@@ -108,7 +108,7 @@
                                             </span>
                                             <t t-set="opened">
                                                 <NotificationPopover
-                                                    notificationLocalIds="messageView.message.notifications.map(notification => notification.localId)"
+                                                    messageLocalId="messageView.message.localId"
```

Comment:
Ok for now, but we'll need to adapt this later of course (both create a model for the notification popover, and use the new popover hook).